app/controllers/sitemap_controller.rb

```diff
@@ -45,6 +45,22 @@ def build
   end
 
   def published_pages
+    content_pages.merge(event_pages)
+  end
+
+  def event_pages
+    events.map { |e| event_path(e.readable_id) }.index_with({})
+  end
+
+  def events
+    GetIntoTeachingApiClient::TeachingEventsApi.new.search_teaching_events(
+      start_after: Time.zone.now,
+      quantity: 100,
```

Comment:
I would prefer a constant (defined in this controller like DEFAULT_LASTMOD), called something like FUTURE_EVENTS_LIMIT rather than a hardcoded number, but that's just my preference.